src/rpc_client/src/compute_client.rs

```diff
@@ -37,15 +39,16 @@ pub struct ComputeClient {
 }
 
 impl ComputeClient {
-    pub async fn new(addr: HostAddr) -> Result<Self> {
+    pub async fn with_address(addr: HostAddr) -> Result<Self> {
         let channel = Endpoint::from_shared(format!("http://{}", &addr))?
+            .initial_connection_window_size(MAX_CONNECTION_WINDOW_SIZE)
```

Comment:
What if we forget this configuration for some other client? 🤣 Should we reuse these in client pool? cc @zwang28

Reply:
Agree.